pipeline-scripts/buildlib.groovy

```diff
@@ -784,16 +784,31 @@ def sync_images(major, minor, mail_list, build_number) {
     }
     def fullVersion = "${major}.${minor}"
     def results = []
-    def metadataJob = fullVersion == '4.6' ? 'build%2Folm_bundle' : 'build%2Fappregistry'
 
-    parallel "build-sync": {
-        results.add build(job: 'build%2Fbuild-sync', propagate: false, parameters:
-            [ param('String', 'BUILD_VERSION', fullVersion) ]  // https://stackoverflow.com/a/53735041
-        )
-    }, metadata: {
-        results.add build(job: metadataJob, propagate: false, parameters:
-            [ param('String', 'BUILD_VERSION', fullVersion) ]  // https://stackoverflow.com/a/53735041
-        )
+    if (fullVersion == '4.6') {
```

Comment:
How about 4.6 beyond? like 4.7 4.8..., if it's the same as 4.6 maybe we can let `if` condition to be fullVersion less than 4.6 and `else` to be the above equal 4.6?

Reply:
Hi @shiywang we still don't know what should happen after 4.6, maybe they won't even be built by ART ... that's why I left anything different from 4.6 to just fall in the "else" block. but we have no idea how it will look like.

Reply:
okay, thanks for explaining to me, I guess we can always correct the course if something changed.